rotkehlchen/chain/solana/node_inquirer.py

```diff
@@ -136,6 +139,24 @@ def get_raw_account_info(self, pubkey: Pubkey) -> bytes:
 
         return response.data
 
+    def get_raw_accounts_info(self, pubkeys: list[Pubkey]) -> dict[SolanaAddress, Account]:
+        """Query the raw accounts info for the given pubkeys.
+        May raise RemoteError if there is a problem with querying the external service.
+        """
+        aggregated_response: dict[SolanaAddress, Account] = {}
+        for chunk in get_chunks(pubkeys, MAX_ACCOUNTS_PER_REQUEST):
+            if (response := self.query(
+                method=partial(lambda client, keys: client.get_multiple_accounts(keys), keys=list(chunk)),  # noqa: E501
+            ).value) is None:
+                raise RemoteError(f'Got empty response for raw solana accounts info for {chunk!s}')
+
+            # response array maintains the same length and order as the input pubkeys.
+            # non-existent accounts return as None in their corresponding positions.
+            # see: https://solana.com/docs/rpc/http/getmultipleaccounts
+            aggregated_response.update({SolanaAddress(str(addr)): info for addr, info in zip(pubkeys, response, strict=False) if info is not None})  # noqa: E501
```

Comment:
That's not a guarantee then. You can't trust external APIs **ever**. What if they break for whatever reason? Then they break our own implementation.

Anything that comes from the outside needs to be verified in code.